lib/LaTeXML/Util/Unicode.pm

```diff
@@ -24,6 +23,73 @@ sub UTF {
+    unwrapped => "`", pos => 'OVERACCENT' },                             #  (OR \x{2035} or UTF(0x60) ?)
+  { name => 'acute', combiner => "\x{0301}", standalone => UTF(0xB4),    # \\'
+    unwrapped => UTF(0xB4), pos => 'OVERACCENT' },                       # (OR \x{2032} or UTF(0xB4)?)
+  { name => 'hat', combiner => "\x{0302}", standalone => "\x{02C6}",     # \^
+    unwrapped => UTF(0x5E), pos => 'OVERACCENT' },
+  { name => 'ddot', combiner => "\x{0308}", standalone => UTF(0xA8),     # \"
+    unwrapped => UTF(0xA8), pos => 'OVERACCENT' },                       # (or \x{22C5})
+  { name => 'tilde', combiner => "\x{0303}", standalone => "\x{02DC}",    # \~
+    unwrapped => UTF(0x7E), pos => 'OVERACCENT' },
+  { name => 'bar', combiner => "\x{0304}", standalone => UTF(0xAF),       # \=
+    unwrapped => UTF(0xAF), pos => 'OVERACCENT' },
+  { name => 'dot', combiner => "\x{0307}", standalone => "\x{02D9}",      # \.
+    unwrapped => "\x{02D9}", pos => 'OVERACCENT' },                       # (OR \x{22C5} or \x{0209} ?
+  { name => 'dtick', combiner => "\x{030B}", standalone => "\x{02DD}",    # \H
+    unwrapped => "\x{2032}\x{2032}", pos => 'OVERACCENT' },               # (Or UTF(0xA8) or " ?)
+  { name => 'breve', combiner => "\x{0306}", standalone => "\x{02D8}",    # \u
+    unwrapped => "\x{02D8}", pos => 'OVERACCENT' },
+  { name => 'check', combiner => "\x{030C}", standalone => "\x{02C7}",    # \v
+    unwrapped => "\x{02C7}", pos => 'OVERACCENT' },
+  { name => 'ring', combiner => "\x{030A}", standalone => "\x{02DA}",     # \r
+    unwrapped => "\x{02DA}", pos => 'OVERACCENT' },                       # (or \x{2218} ?)
+  { name => 'vec', combiner => "\x{20D7}", standalone => $NBSP . "\x{20D7}",    # \vec
+    unwrapped => "\x{2192}", pos => 'OVERACCENT' },
+  { name => 'tie', combiner => "\x{0361}", standalone => $NBSP . "\x{0361}",    # \t
+    unwrapped => "u", pos => 'OVERACCENT' },
+  ## UNDERACCENT accents
+  { name => 'cedilla', combiner => "\x{0327}", standalone => UTF(0xB8),         # \c
+    unwrapped => UTF(0xB8), pos => 'UNDERACCENT' },                             # not even math?
+  { name => 'underdot', combiner => "\x{0323}", standalone => '.',              #  \@text@daccent
+    unwrapped => "\x{22C5}", pos => 'UNDERACCENT' },                            # (Or \x{02D9} ?)
+  { name => 'underbar', combiner => "\x{0331}", standalone => '_',
+    unwrapped => UTF(0xAF), pos => 'UNDERACCENT' },
+  { name => 'lfhook', combiner => "\x{0326}", standalone => ",",                # '\lfhook'
+    unwrapped => ',', pos => 'UNDERACCENT' },
+  { name => 'ogonek', combiner => "\x{0328}", standalone => "\x{02DB}",
+    unwrapped => "\x{02DB}", pos => 'UNDERACCENT' },                            # not even math???
+);
+# Set up a hash keyed on both standalone & combiner chars
+our %accent_data = ();
+foreach my $entry (@accent_data) {
+  $accent_data{ $$entry{standalone} } = $entry;
+  $accent_data{ $$entry{combiner} }   = $entry;
+}
```

Comment:
Oh, this is rather dangerous. Could you change the name of either `@accent_data` or `%accent_data`?

I was thrown for a bit not knowing what happened, seeing the hash indexing syntax in `$accent_data{$char}`. 

Such a setup is bound to generate a subtle bug one day, when someone uses the wrong delimiters in haste:
```perl
my @a = ('x','y','z');
my %a = ('1'=>'a', '2'=>'b', '3'=>'c');
print $a[2]; # z
print $a{2}; # b
```

Reply:
I think the hash variant can be kept, and you can even fully expand the loop that adds `standalone` and `combiner` in code, to avoid looping the setters every time latexml is ran...

Simplest is to run it once, print with Dumper, then copy the result back into the file. And on second read - aren't `standalone` and `combiner` already set in the explicit hashes? Maybe the loop is a leftover? Except that the loop sets them as equal, while the hash often has the values as different. Hm, I wish I understood a little more...

Reply:
Renamed a bit to make it less scary.